addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -106,7 +106,10 @@ def _prepare_invoice_values(self, order, name, amount, so_line):
     def _get_advance_details(self, order):
         context = {'lang': order.partner_id.lang}
         if self.advance_payment_method == 'percentage':
-            amount = order.amount_untaxed * self.amount / 100
+            if self.product_id.taxes_id.price_include:
+                amount = order.amount_total * self.amount / 100
+            else:
+                amount = order.amount_untaxed * self.amount / 100
```

Comment:
If there are multiple taxes on the down payment product, this won't work.
![image](https://user-images.githubusercontent.com/19364378/106449600-6a7d7680-6484-11eb-84bb-dc03322c0adc.png)

Reply:
Thanks for the note, this was taken care of in the fw port

Reply:
But the problem still exists in 13.0 since you only modified the fw-ports :c.